app/presenters/concerns/validation_tasks/documents_presenter.rb

```diff
@@ -4,6 +4,7 @@ module ValidationTasks
   extend ActiveSupport::Concern
 
   class DocumentsPresenter < PlanningApplicationPresenter
```

Comment:
I've removed this class in [this change](https://github.com/unboxed/bops/pull/880/files#diff-342eeb61b06bda5c910d2bd51dcd8cf85fec5ef02a340f3a2a3404029892c9d9)!

I think easiest if you merge this first, and then I'll do a careful rebase ☺️